.github/workflows/integration-tests.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         run: .github/scripts/protoc.sh
         shell: bash
       - name: Build
-        run: cargo build --features consensus
```

Comment:
Given that the binaries are now equivalent, we could also just move `./tests/integration-tests.sh distributed` as an additional step to the previous job to not compile twice.